bootstrap/services/storm-bundle.json

```diff
@@ -4,33 +4,62 @@
   "serviceUISpecification": {
     "fields": [
       {
+        "uiName": "Nimbuses hostnames",
```

Comment:
Storm uses `Nimbuses` and `Nimbodes` to represent plural form of Nimbus.
I'm not clear which word Nathan referred, but at least one of word `nimbus` has `nimbuses` as plural form.
https://www.merriam-webster.com/dictionary/nimbus